aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -120,19 +120,21 @@ MPSDataType getMPSScalarType(ScalarType scalar_type) {
   switch (scalar_type) {
     case ScalarType::Double:
     case ScalarType::Float:
-      return "MPSDataTypeFloat32";
+      return "Float32";
```

Comment:
These are strings for MPS data types. We don't print these anywhere in debug correct ?

Reply:
Yes, they're used in keys only.